web-packages/test/case/2d/src/common/test-player.ts

```
@@ -0,0 +1,230 @@
+      const span = document.createElement('span');
+
+      div?.appendChild(span);
+    }
+  }
+
+  clearResource () {
+    this.player.destroyCurrentCompositions();
+  }
+
+  disposeScene () {
+    if (this.composition && this.composition.dispose) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)